packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -1,12 +1,23 @@
 import { deployContract, deployer, exportDeployments } from "./deploy-contract";
 
 const deployScript = async (): Promise<void> => {
-  await deployContract(
-    {
-      owner: deployer.address, // the deployer address is the owner of the contract
-    },
-    "YourContract"
-  );
+  try {
+    await deployContract(
+      {
+        owner: deployer.address, // the deployer address is the owner of the contract
+      },
+      "YourContract"
+    );
+  } catch (error) {
+    console.log(error.message);
+    if (error.message.includes("fetch failed")) {
+      console.error(
+        "Couldn't deploy. Did you forget to run yarn chain? Or you can change target network in scaffold.config.ts"
+      );
+    }
+
+    throw error;
+  }
 };
```

Comment:
lets not do the warning here, we should leave this error logging for another issue that covers all types of errors that can come on deployment.